perf/bench/r2c-rules/r2c-ci.yml

```diff
@@ -577,8 +577,6 @@ rules:
 - id: java.lang.correctness.no-string-eqeq.no-string-eqeq
   languages:
   - java
-  equivalences:
```

Comment:
To be clear, does this equivalence now happen automatically?

Reply:
I didn't think so but https://github.com/returntocorp/semgrep-rules/pull/1370/files makes it seem like it does.